comp/core/gui/guiimpl/platform_nix.go

```diff
@@ -11,10 +11,22 @@ import (
 	"fmt"
 )
 
+const docURL = "https://docs.datadoghq.com/agent/basic_agent_usage/?tab=Linux"
+
 func restartEnabled() bool {
 	return false
 }
 
 func restart() error {
 	return fmt.Errorf("restarting the agent is not implemented on non-windows platforms")
 }
+
+func logginInstructions() string {
```

Comment:
We should probably use `html/template` here, it would automatically encode characters from the URL if needed, which is a simple way to avoid errors in the future

Reply:
+1. Also ditto on function name from earlier comment from Pierre.

Reply:
Right ! I replaced the main template in `gui.go` (made in 0eab592fdb645d283db60f88460c62ab59ab1ed9)